packages/material-ui-system/src/GlobalCss/colors.js

```diff
@@ -0,0 +1,26 @@
+import combineWithBreakpoints from './combineWithBreakpoints';
+
+const r = (val, accumulator, colors) => {
+  if (typeof val === 'string') {
+    colors[`bg-${accumulator}`] = { backgroundColor: `${val} !important` };
```

Comment:
I had to add !important here to make sure that the classes would override the default styles, not sure if it is the best approach.. FYI @oliviertassinari

Reply:
Please don't use any `!important`

Reply:
Fixed, with the index option on `withStyles` 👍